Permanent/ViewControllers/AccountSettingsViewController.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  AccountSettingsViewController.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 07.01.2021.
+//  Copyright © 2021 Victory Square Partners. All rights reserved.
+//
+
+import UIKit
+
+class AccountSettingsViewController: BaseViewController<SecurityViewModel> {
+
+    
+    
+    @IBOutlet weak var contentUpdateButton: RoundedButton!
+    @IBOutlet weak var currentPasswordView: PasswordElementView!
+    @IBOutlet weak var reTypePasswordView: PasswordElementView!
+    @IBOutlet weak var newPasswordView: PasswordElementView!
+    @IBOutlet weak var firstLineView: UIView!
+    @IBOutlet weak var logWithFaceIdView: SwitchSettingsView!
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        viewModel = SecurityViewModel()
+        
+        self.title = .security
+        view.backgroundColor = .white
+        firstLineView.backgroundColor = .lightGray
+        logWithFaceIdView.text = .logInWithFingerPrint
+        
+        contentUpdateButton.configureActionButtonUI(title: .updatePassword)
+        
+        currentPasswordView.configurePasswordElementUI(label: .currentPassword, returnKey: UIReturnKeyType.next)
+        reTypePasswordView.configurePasswordElementUI(label: .reTypePassword, returnKey: UIReturnKeyType.done)
+        newPasswordView.configurePasswordElementUI(label: .newPassword , returnKey: UIReturnKeyType.next)
+        
+        self.currentPasswordView.delegate = self
+        self.reTypePasswordView.delegate = self
+        self.newPasswordView.delegate = self
+        self.logWithFaceIdView.delegate = self
+
+        let authStatus = PermanentLocalAuthentication.instance.canAuthenticate()
```

Comment:
We should move the business logic inside the viewmodel.

And the viewmodel will tell us the state that the ui should be.

Reply:
Moved authStatus bussiness logic in viewmodel.